pkg/network/network.go

```diff
@@ -56,6 +67,25 @@ type Config struct {
 
 	// DefaultClusterIngressClass specifies the default ClusterIngress class.
 	DefaultClusterIngressClass string
+
+	// RouteTemplateText is the golang text template used to generate the
+	// Route's URL for the Service. This is the raw text string provided
+	// in the Network ConfigMap.
+	RouteTemplateText string
```

Comment:
Do we really need that?

Reply:
We use this value in the error message in https://github.com/knative/serving/pull/3306/files#diff-001ba74612c5f969aa98bfefb45fbff9R448 so that we can show the value/text that couldn't be processed.